src/views/HomePage.jsx

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import NavBar from './Navbar';
+import { Card, BuzzSection } from '../components';
+import { getArticles } from '../actions/homeActions';
+import Footer from '../components/Footer';
+import Spinner from '../components/Spinner';
+
+class HomePage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      articles: null,
+    };
+    this.getArticles = props.getArticlesAction;
+  }
+
+  componentWillMount = async () => {
+    const { getArticlesAction } = this.props;
+    const articles = await getArticlesAction();
+    this.setState({ articles: articles.data });
+  };
+
+  bookmark = (event) => {
+    event.target.classList.add('bookmark-clicked');
+  };
+
+  like = (event) => {
+    event.target.classList.add('like-clicked');
+  };
+
+  bookmark = (event) => {
+    event.target.classList.toggle('bookmark-clicked');
+  };
+
+  like = (event) => {
+    event.target.classList.toggle('like-clicked');
+  };
+
```

Comment:
I'm not sure why we have the functions `bookmark` and `like` repeated. I think one of each pair should a removed.
<img width="1029" alt="screen shot 2019-01-26 at 9 33 51 pm" src="https://user-images.githubusercontent.com/38390690/51792515-275fdd00-21b2-11e9-8b0c-632ae30567f3.png">